packages/console/src/components/ToggleTipButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+import classNames from 'classnames';
+import type { ReactElement } from 'react';
+import { useRef, useState } from 'react';
+
+import Tip from '@/assets/images/tip.svg';
+import type { HorizontalAlignment } from '@/hooks/use-position';
+import { onKeyDownHandler } from '@/utilities/a11y';
+
+import type { TipBubblePosition } from '../TipBubble';
+import ToggleTip from '../ToggleTip';
+import * as styles from './index.module.scss';
+
+type Props = {
+  render: (closeTipHandler: () => void) => ReactElement;
+  className?: string;
+  tipPosition?: TipBubblePosition;
+  tipHorizontalAlignment?: HorizontalAlignment;
+};
+
+const ToggleTipButton = ({ render, className, tipPosition, tipHorizontalAlignment }: Props) => {
+  const anchorRef = useRef<HTMLDivElement>(null);
+  const [isTipOpen, setIsTipOpen] = useState(false);
+
+  const closeTipHandler = () => {
+    setIsTipOpen(false);
+  };
+
+  return (
+    <div className={classNames(styles.toggleTipButton, className)}>
+      <div
+        ref={anchorRef}
+        role="tab"
+        tabIndex={0}
+        className={styles.icon}
+        onClick={() => {
+          setIsTipOpen(true);
+        }}
+        onKeyDown={onKeyDownHandler(() => {
+          setIsTipOpen(true);
+        })}
+      >
+        <Tip />
+      </div>
+      <ToggleTip
```

Comment:
wondering if the ToggleTip is being used by any other places instead of the ToggleTipButton. If not better move them under the same component folder like ToggleTipButton, {ToggleTip }

Reply:
Not sure, just want to keep aligned with the Web Design System, for we have `ToggleTip` and `Tooltip`, and prefer to keep them clean and tidy and can be used to construct other components.

Reply:
Add an issue to classify AC components: LOG-4864